### PR TITLE
Added date range input to graphs

### PR DESCRIPTION
**Pull request by adamsr123**

### CODE REVIEW

File: src/components/User/AlufStats/DateRangeInput.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        <div>
+            <Typography variant="h6" display="block" className={dateTitles}>
+                בחר טווח תאריכים
+            </Typography>
+            <div className={dateTimePickers}>
+                <div className={rightPicker}>
+                    <KeyboardDatePicker
+                        margin="normal"
+                        required
+                        label="בחר תאריך"
```
Comment (amitaiblevy) [inline src/components/User/AlufStats/DateRangeInput.js:35]: should be clear which is start and which is end. בחר תאריך התחלה.

File: src/components/User/AlufStats/DateRangeInput.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    },
+    dateTimePickers: {
+        display: "flex"
+    },
+    dateTitles: {
+        marginTop: theme.spacing(3)
+    }
+}));
+
+export default function DateRangePicker({ startDate, endDate,
```
Comment (amitaiblevy) [inline src/components/User/AlufStats/DateRangeInput.js:21]: props should be time -{start, end} and setTime. validation of startDate < endDate should occur in here before call of onChange

File: src/components/User/AlufStats/DateRangeInput.js (hunk 1 above)
Comment (adamsr123) [reply src/components/User/AlufStats/DateRangeInput.js]: fixed

File: src/components/User/AlufStats/DateRangeInput.js (hunk 2 above)
Comment (adamsr123) [reply src/components/User/AlufStats/DateRangeInput.js]: fixed

**Pull request merged**